.github/workflows/nightly_release.yml

```diff
@@ -88,7 +105,7 @@ jobs:
         with:
           payload: |
             {
-              "test": "Successfully released PWA Kit v${{ env.nightly_version }}"
+              "message": "Successfully released PWA Kit v${{ env.nightly_version }}"
```

Comment:
Original slack-webhook message body was configured to run only for failures. I updated the configuration to make it more generic.